src/validator.js

```diff
@@ -412,6 +412,29 @@ export default class Validator
     }
 
     /**
+     * Append another validation to an existing field.
+     *
+     * @param  {string} name The field name.
+     * @param  {string} checks validations expression.
+     */
+    append(name, checks, options = {}) {
+        // No such field
+        if (!this.$fields[name]) {
+            return;
+        }
+
+        const checksArray = checks.split('|').map(rule => {
+            const normalizedRule = this._normalizeRule(rule, this.$fields[name].validations);
+            if (! normalizedRule.name) {
+                return;
```

Comment:
This would cause undefined checks, which isn't a huge problem but I think we should keep the validations array for each field as clean as possible. perhaps use forEach instead and collect them in an external array.